upload-artifact.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+set -u
+
+VERSION=$(grep "^version" ./threedi_api_qgis_client/metadata.txt | cut -d= -f2)
+
+# ARTIFACTS_KEY should be set as env variable in the travis UI.
+# TRAVIS_BRANCH is set automatically by travis
+ARTIFACT=threedi_api_qgis_client_${VERSION}.zip
+PROJECT=threedi_api_qgis_client
```

Comment:
De rest van de projecten heeft streepjes:

```suggestion
PROJECT=threedi-api-qgis-client
```

Het heeft verder geen effect ergens op (behalve consistentie).